dotnet-base-libraries/ReturnInsertedIdentityWithDapper/ReturnInsertedIdentityWithDapper/Methods/UseOfDapperExtension.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Dapper.Contrib.Extensions;
+using Microsoft.Data.SqlClient;
+
+namespace ReturnInsertedIdentityWithDapper;
+
+public partial class Methods
```

Comment:
Again, file is called `UseOfDapperExtensions` while the class is called `Methods` - it should be consistent

Reply:
Renamed file to Methods